src/main/java/frc/robot/subsystems/internals/InternalSubsystem.java

```diff
@@ -143,14 +138,12 @@ public void periodic() {
             motorBottom.set(0);
 
             prevEntranceDetected = false;
```

Comment:
Does the explicit set of  `prevEntranceDetected` to false accomplish anything different when it's set to `entranceDetected` on line 110? I feel like this set would just be immediately overridden the next periodic loop.

Reply:
I believe it allows us to detect if a new ball enters or is entering internals, though it probably is already false from the ball moving. We can delete it, especially if it might be true (and another ball has entered).